Kconfig

```
@@ -2067,6 +2067,10 @@ menu "LVGL configuration"
 	endmenu
 
 	menu "Demos"
+		config LV_BUILD_DEMOS
+			bool "Build demos"
+			default y if !LV_CONF_MINIMAL
+
 		config LV_USE_DEMO_WIDGETS
 			bool "Show some widget"
```

```suggestion
	menu "Demos"
		config LV_BUILD_DEMOS
			bool "Build demos"
			default y if !LV_CONF_MINIMAL

		if LV_BUILD_DEMOS

		config LV_USE_DEMO_WIDGETS
			bool "Show some widget"
```

With `endif` and the end.

---

Agreed and done. Thanks :)